contrib/storage-hbase/src/main/resources/drill-module.conf

```diff
@@ -17,6 +17,11 @@
 //  This file can also include any supplementary configuration information.
 //  This file is in HOCON format, see https://github.com/typesafehub/config/blob/master/HOCON.md for more information.
 
+drill.classpath.scanning: {
```

Comment:
Can you put this in single drill block? E.g.

drill: {
  classpath.scanning: ...,
  exec: {
   ... existing exec block
  }
}